### PR TITLE
Update id.coffee

### PR DESCRIPTION
**Pull request by Agalondia**
Here's my translation for Indonesian, please consider it guys!

If i forgot any sentence in here, just tell me. I'll translate the rest of them tomorrow, or before i prepare for school.

### CODE REVIEW

**Comment by Imperadeiro98** [comment]
Hello, @Agalondia! Thank you for contributing. Can you sign our [CLA](https://codecombat.com/cla) so that I can merge your pull request?

Can you also uncomment te strings you translated (remove the # symbol)?

**Comment by Agalondia** [comment]
@Imperadeiro98 
> Can you also uncomment te strings you translated (remove the # symbol)?

Uhhhhhh i have only edited  the part inside the quote mark, and i don't which # you're talking about honestly.

**Comment by Agalondia** [comment]
> Can you also uncomment te strings you translated (remove the # symbol)?

Uhhhhhh i have only edited the part inside the quote mark, and i don't which # you're talking about honestly.

File: app/locale/id.coffee [hunk 1]
```diff
@@ -1,108 +1,108 @@
 module.exports = nativeDescription: "Bahasa Indonesia", englishDescription: "Indonesian", translation:
   home:
-#    slogan: "Learn to Code by Playing a Game"
-    no_ie: "Maaf, CodeCombat tidak bisa dijalankan pada Internet Explorer 8 dan sebelumnya" # Warning that only shows up in IE8 and older
+#    slogan: "Belajar Membuat Kode Dengan Bermain Sebuah Permainan"
```
Comment (Imperadeiro98) [inline app/locale/id.coffee]: For example, in this string you would remove the # symbol from the beginning of the sentence:

from
`# slogan: "Belajar Membuat Kode Dengan Bermain Sebuah Permainan"`

to
`slogan: "Belajar Membuat Kode Dengan Bermain Sebuah Permainan"`

**Comment by Imperadeiro98** [comment]
Thanks

**Pull request merged**